docs/05-terraform.md

```diff
@@ -15,4 +15,192 @@ This Terraform script can be used to deploy an entire environment from scratch.
+
+### Terraform import block Template
+We have created a Terraform import block template in `terraform-deployment/importtemplate.md`. In this template, you can select the needed import block and paste it into the `imports.tf` file within the `terraform-deployment/deployments/imports.tf` directory. Remember to change the ID to the referring ID of your existing infrastructure.
+
+## Managing Your Infrastructure
+
+Once your environment is up and running, you can use this configuration to manage it:
+
+1. Code Updates: If you make any code changes in `TorQ` or `TorQ-Amazon-FinSpace-Starter-Pack` and want to apply these to your clusters, rezip these directories and run the Terraform deployment again. This will recreate clusters with the updated code.
+2. Cluster Config: If you want to make changes to a cluster's config settings (e.g., node size of the RDB), update this in `clusters/rdb.tf` and run Terraform again. The RDB will be recreated with the new node size.
+3. Delete/Create Clusters: Clusters can be deleted or created individually or all at once from the `terraform.tfvars` file. To delete a cluster, set its count to 0. To delete all clusters, set `create-clusters` to 0.
+4. log groups and metric filters: These resources are only created if the dependent log groups exists. Update the `wdb_log_groups` variable in `terraform.tfvars` to include the names of the log groups of your clusters you wish to monitor. WARNING: this terraform stack will fail inelegantly if you list a name of an unexisting log group. To be amended in future iterations.
+
+
+### Basic Commands in Terraform 
+*  `terraform init`       -   Prepare your working directory for other commands
+*  `terraform validate`   -   Check whether the configuration is valid
+*  `terraform plan`       -   Show changes required by the current configuration
+*  `terraform apply`      -   Create or update infrastructure
+*  `terraform destroy`    -   Destroy previously-created infrastructure
+* For more commands in Terraform, please visit [Terraform Command](https://developer.hashicorp.com/terraform/cli/commands) 
+
+### Terraform State Management
+
+Terraform maintains a state file that tracks the state of the deployed infrastructure. This state file is crucial for Terraform to understand what resources have been created and to make changes to them. To ensure proper state management:
+
+- Always store your state files securely, as they may contain sensitive information.
+- Consider using remote state storage, such as Amazon S3, to keep your state files safe and accessible from multiple locations.
+- Avoid manual changes to resources managed by Terraform, as this can lead to inconsistencies between the actual infrastructure and Terraform's state.
+
+## List of AWS Structures that will be created with our Terraform deployment
```

Comment:
I think a number of these structures will not be included in the MVP. Will update this comment when I know more details